src/blenderbim/blenderbim/bim/module/classification/operator.py

```diff
@@ -54,9 +55,9 @@ def _execute(self, context):
         ifcopenshell.api.run(
             "classification.add_classification",
             IfcStore.get_file(),
-            **{"classification": Data.library_file.by_id(int(props.available_classifications))},
+            **{"classification": ClassificationsData.data["library_file"].by_id(int(props.available_classifications))},
         )
-        Data.load(IfcStore.get_file())
+        ClassificationsData.load()
```

Comment:
The operator should not call the load() function. Instead, the operators should inherit from an `Operator` class like so: https://github.com/IfcOpenShell/IfcOpenShell/blob/v0.7.0/src/blenderbim/blenderbim/bim/module/owner/operator.py#L27-L31 - then you can also delete call def execute().